packages/client/rollup.config-1738965447747.mjs

```
@@ -0,0 +1,246 @@
+    runnerRoot,
+    previousAnalysisDir,
+    relativePath,
+    analysisFilePath,
+  );
+  return {
+    currentPath,
+    prevPath,
+  };
+}
+const ansiRegex = new RegExp(
+  '([\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)|(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-ntqry=><~])))',
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)